.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -63,6 +63,11 @@
     <MonoAndroidAssetsPrefix>$(AndroidProjectFolder)Assets</MonoAndroidAssetsPrefix>
   </PropertyGroup>
 
+  <!-- .NET Standard or .NET 6 -->
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
```

Comment:
There should be no need to re-define these (https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md)

Reply:
I am cheating here. The net6.0 TFM does not define NETSTANDARD because it is not, but since maui treats net6 and netstd the same, I am pretending that net6.0 is netstd.